web/src/index.tsx

```diff
@@ -1,19 +1,36 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
```

Comment:
Can you standardise quotes + formatting for the projects in one PR, this reformatting clutters the PR changes when reviewing